### PR TITLE
Update Genesis Slot and Epoch

### PR DESCRIPTION
**Pull request by terenc3t**
This is part of #1541  

Key changes:
- genesis slot to 2^63
- genesis epoch to 2^63/64
- fixed references and tests

### CODE REVIEW

File: beacon-chain/chaintest/tests/state-tests/block-processing.yaml [hunk 1]
```diff
@@ -4,35 +4,35 @@ test_suite: prysm
...
-    results:
-      slot: 32
-      num_validators: 5000
-  - config:
-      epoch_length: 63
       deposits_for_chain_start: 16384
       num_slots: 64 # Testing advancing state to exactly slot == EpochLength
       deposits:
-        - slot: 1
+        - slot: 9223372036854775809
```
Comment (terenc3t) [inline beacon-chain/chaintest/tests/state-tests/block-processing.yaml]: haven't spent much time to research, I wonder if there's an elegant way to represent this, yams won't let me do + or power notations

Comment (rauljordan) [reply beacon-chain/chaintest/tests/state-tests/block-processing.yaml]: id say use a string @terenc3t

**Comment by rauljordan** [comment]
Tests still failing @terenc3t

**Comment by terenc3t** [comment]
This one is tricky with chain test, everything is tightly coupled slot being 0 😞

**Comment by terenc3t** [comment]
> Tests still failing @terenc3t

@rauljordan this is ready

**Pull request merged**